stdlib/public/core/Array.swift

```diff
@@ -646,31 +647,6 @@ extension Array {
     return _buffer.capacity
   }
 
-  /// - Precondition: The array has a native buffer.
-  @inlinable
-  @_semantics("array.owner")
```

Comment:
Can we remove optimizer code that checks for this semantics attribute also?

```
palomino:swift spestov$ git grep kGetArrayOwner
include/swift/SILOptimizer/Analysis/ArraySemantic.h:  kGetArrayOwner,
lib/SILOptimizer/Analysis/ArraySemantic.cpp:            .Case("array.owner", ArrayCallKind::kGetArrayOwner)
lib/SILOptimizer/Analysis/EscapeAnalysis.cpp:      case ArrayCallKind::kGetArrayOwner:
lib/SILOptimizer/LoopTransforms/COWArrayOpt.cpp:  case ArrayCallKind::kGetArrayOwner:
lib/SILOptimizer/LoopTransforms/COWArrayOpt.cpp:  case ArrayCallKind::kGetArrayOwner:
```

Reply:
It's still there in `ContiguousArray` and `ArraySlice`, which are next up.